### ISSUE TITLE
aem 'import' should allow a branch

### ISSUE THREAD

**Issue by atopper** [adobe/helix-cli#2426]
**Description**
Switch to a development or new feature branch is awkward when running `aem import`.  The tests imply that it can be done through the `ui-repo` parameter (i.e. **_https://github.com/adobe/helix-importer-ui#somebranch_**) but that does not work.

This would be particularly useful when trying to socialize a new feature or bug fix, and reduce the effort to try it.

**To Reproduce**
1. Be able to run `aem import` successfully
2. Pick a branch in https://github.com/adobe/helix-importer-ui/branches, say 'test'
3. Try to start the importer with that branch: `aem import --ui-repo https://github.com/adobe/helix-importer-ui#test`)
4. Start up fails

**Expected behavior**
The Helix importer UI should start up and change branches to the specified test

**Screenshots**
N/A

**Version:**
Around 16.7.5

**Additional context**
The use of the fragment in the UI isn't publicly advertised, so could be considered a feature.

**Comment by kptdobe** [comment]
You can do that ("manually"):
1. run `aem import`
2. go to `./tools/importer/helix-importer-ui`
3. `git checkout <branch>`
4. build the current branch: `npm run build`
5. reload the importer UI

**Comment by atopper** [comment]
Yes @kptdobe, that's true.  And then the user can add the `--skip-ui` to prevent checking out main again.

I find it is challenging to get people to try a new branch, and if we made it easy and reduce the steps by using _ui-repo_, perhaps even sending them the exact command line to run (i.e. `aem import --ui-repo=https://github.com/adobe/helix-importer-ui#test`), it would be easier for them to quickly try it, and to continue to use.
So it would mainly be for convenience, but I think a good convenience.

**Comment by arumsey** [comment]
I agree with @atopper that the "manual" approach adds too much friction to provide our customers with early access to new features. The aem cli should make it incredibly easy for users of the AEM Importer to access feature branches. The downside would be that it would be too easy to access branches that should not be used and it could encourage the proliferation of even more AEM Importer variations instead of simply focusing on integrating features into the core product.

**Comment by kptdobe** [comment]
I did not realise we already had the `ui-repo` property. Not supporting branches make it useless.

**Comment by kptdobe** [comment]
Fixed with #2432
